pegasus/sites.v3/code.org/public/css/starwars.css

```diff
@@ -100,6 +104,12 @@
   }
 }
 
+@media screen and (min-width: 500px) and (max-width: 600px) {
```

Comment:
Is it worth shrinking this font for more widths, rather than for such a specific range?

Reply:
I think going back to a larger font when it's super small works well because there's more space. This specific range is kind of unique to this copy and some of the ranges already set for the box widths.

| Dimensions, Font size | Image |
| ---------------------- | ------ |
| 320 - 499px, 17px | ![image](https://user-images.githubusercontent.com/1920530/47836439-88b81180-dd65-11e8-9af1-d032c34fe41d.png) |
| 500 - 512px, 14px | ![image](https://user-images.githubusercontent.com/1920530/47836510-d3d22480-dd65-11e8-9383-9898b3dce1ff.png) | 
| 513 - 600px, 14px | ![image](https://user-images.githubusercontent.com/1920530/47836523-e0ef1380-dd65-11e8-9fce-dc688961661f.png) | 
| 600 - 970px, 17px | ![image](https://user-images.githubusercontent.com/1920530/47836571-0d0a9480-dd66-11e8-87e0-64e9293631e8.png) |
| 970px+, 22px | ![image](https://user-images.githubusercontent.com/1920530/47836587-227fbe80-dd66-11e8-9dde-fad31c3c69e7.png) |